Source/Resources/Content/MSGraph/DeviceManagement/GroupPolicyConfigurations/Functionality - Microsoft Edge Configuration.json

```diff
@@ -91,5 +91,5 @@
       "presentationValues": []
     }
   ],
-  "description": ""
+  "description": "What is it: Configures Microsoft Edge for all devices to sign into the browser and use Google as the default search engine. \n Simeon Notes: The Chromium version of Edge is surprisingly a good browser. This helps make it even better by allowing users to have their history and passwords tied to their account."
```

Comment:
**What does this do?**
Configures Microsoft Edge for all devices to automatically sign into the browser and synchronize history and passwords. Also sets Google as the default search engine. 

**Why should you use this?**
To enable an optimal user experience for the new Chromium Edge browser.

**What is the end-user impact?**
User's search history and passwords will be synchronized across devices.

**Learn more**
N/A